.github/workflows/build.yaml

```diff
@@ -21,7 +21,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-cargo-
       - name: build
-        run: make
+        run: |
+          make
+          mkdir ./bin
+          cp containerd-shim-slight-v1/target/release/containerd-shim-*-v1 bin/
```

Comment:
Since different shims are not sharing the same workspace, their compiled binaries will not be placed in the same /target/release folder. The build pipeline will create a `bin` folder and place all the binaries inside. This will later be consumed by the release pipeline.